### ISSUE TITLE
Don't send mails to anonymous users

### ISSUE THREAD

**Issue by peteruithoven** [rikva/voko#82]
We don't yet check if the user that we send reminder mails to is still active / not sleeping / not anonymized. Apparently we recently send a mail to `621@anon.vokoutrecht.nl` which failed. 

We send these mails from for example the transport and distribution schedules.

**Comment by rikva** [comment]
That is really interesting... sleeping members should be automatically filtered out 🤔

**Comment by peteruithoven** [comment]
Are you sure? 

This is the relevant logic:
https://github.com/rikva/voko/blob/5cee46e860bdbf9a23715cc505d6811672561ee4/webapp/ordering/models.py#L289-L304

I just assumed I needed to add an extra check there.

**Comment by mgvd** [comment]
Non-active users are filtered out. But sleeping members are not automatically set to not active.
